src/main/java/meditracker/dailymedication/DailyMedicationManager.java

```diff
@@ -0,0 +1,320 @@
+        DailyMedication dailyMedication = DailyMedicationManager.getDailyMedication(listIndex, Period.MORNING);
+        dailyMedication.untake();
+        try {
+            FileReaderWriter.saveDailyMedicationData(DailyMedicationManager.getDailyMedicationStringData());
+        } catch (FileReadWriteException e) {
+            throw new FileReadWriteException("IO Error: Unable to write to text File");
+        }
+    }
+
+    /**
+     * Takes each DailyMedication object and adds to string of each object
+     * to a list of String and return
+     *
+     * @return A list of DailyMedication object to string
+     */
+    public static List<String> getDailyMedicationStringData() {
+        List<String> dailyMedicationStrings = new ArrayList<>();
+        for (DailyMedication morningMedication : morningMedications) {
+            dailyMedicationStrings.add("M|" + morningMedication.isTaken() + "|" + morningMedication.getName());
+        }
+        for (DailyMedication afternoonMedication : afternoonMedications) {
+            dailyMedicationStrings.add("A|" + afternoonMedication.isTaken() + "|" + afternoonMedication.getName());
+        }
+        for (DailyMedication eveningMedication : eveningMedications) {
+            dailyMedicationStrings.add("E|" + eveningMedication.isTaken() + "|" + eveningMedication.getName());
+        }
+        return dailyMedicationStrings;
+    }
+
+    /**
+     * Returns the total number of daily medications in the list.
+     *
+     * @return The total number of daily medications.
+     */
+    public static int getTotalDailyMedication() {
+        return dailyMedications.size();
+    }
+
+    /**
+     * Checks if added medication is to be taken at morning/afternoon/evening
+     *
+     * @param medication list of medications from MedicationManager
+     */
+    public static void checkForDaily(Medication medication) {
+        if (shouldAddToDailyList(medication)) {
+            addToSubLists(medication);
+        }
+    }
+
+    /**
+     * Returns boolean if medication can be added to today's daily list based on the repeat setting
+     *
+     * @param medication list of medications from MedicationManager
+     * @return true if medication can be added to today's list
+     */
+    private static boolean shouldAddToDailyList(Medication medication) {
+        Repeat repeat = Repeat.repeatDays(medication.getRepeat());
+
+        switch (repeat) {
+        case DAILY:
+            return true;                       //everyday
+        case EVERY_OTHER_DAY:
+            return isNDay(medication, 2); //every 2 days
+        case EVERY_3_DAYS:
+            return isNDay(medication, 3); //every 3 days
+        case EVERY_4_DAYS:
+            return isNDay(medication, 4); //every 4 days
+        case EVERY_WEEK:
+            return isNDay(medication, 7); //every 7 days
+        default:
+            return false;
+        }
+    }
```

Comment:
It might be worth reconsidering the use of the Enum in this context. It seems like you're converting a number to an Enum only to immediately convert it back to a number in order to pass it to `isNDay`. This extra step could potentially be simplified.

Reply:
Thx for the suggestion, removed enum in 63969bc